core/base/Module.php

```diff
@@ -178,6 +178,16 @@ public function getUrlRules()
      */
     public $moduleLayout = 'layout';
 
+    /**
+     * @var string The default file of layout
+     */
+    public $layout = 'main';
+
+    /**
+     * @var string The default folder of theme load
+     */
+    public $theme;
```

Comment:
The theme should be configured via the component, so i think the path for the layout should be extract from the themes component.